modules/installation-guide/partials/proc_manually-upgrading-che-in-operatorhub.adoc

```diff
@@ -5,30 +5,36 @@
 [id="manually-upgrading-che-in-operatorhub_{context}"]
 
 = Manually upgrading {prod-short} in OperatorHub
+This section describes how to upgrade {prod-short} by using the OperatorHub with the `Manual` approval strategy.
+
 .Prerequisites
 
-* An administrator account on an OpenShift instance.
+* An administrator account on an instance of OpenShift.
 
-* An instance of an earlier minor version of {prod-short}, installed using the Operator from OperatorHub on the same instance of OpenShift.
+* An instance of an earlier minor version of {prod-short}. You have installed the instance by using the Operator.
 
-* The approval strategy in the subscription is set to `Manual`.
+* The approval strategy in the subscription is `Manual`.
 
 .Procedure
 
 . Open the OpenShift web console.
 
-. Navigate to the *Operators* -> *Installed Operators* section.
+. Navigate to the *Operators* -> *Installed Operators* page.
+
+. In the list of the installed Operators, click *{prod}*.
+
+. Navigate to the *Subscription* tab. 
 
-. Click *{prod}* in the list of the installed Operators.
+. Next to the *Upgrade Status*, inspect the upgrades that require approval. The expected message is *1 requires approval*.
 
-. Navigate to the *Subscription* tab. Upgrades requiring approval are displayed next to *Upgrade Status*, for example *1 requires approval*.
+. Click *1 requires approval*.
 
-. Click *1 requires approval*, then click *Preview Install Plan*.
+. Click *Preview Install Plan*.
 
-. Review the resources that are listed as available for upgrade and click *Approve*.
+. Review the resources that are available for upgrade and click *Approve*.
 
 .Verification steps
 
-. Navigate to the *Operators* → *Installed Operators* page to monitor the progress of the upgrade. When complete, the status changes to *Succeeded* and *Up to date*.
+. Navigate to the *Operators* -> *Installed Operators* page to monitor the upgrade progress. When complete, the status changes to *Succeeded* and *Up to date*.
 
-. The {prod-ver} version number is visible at the bottom of the page.
+. The {prod-ver} version number is visible at the end of the page.
```

Comment:
Is there a way to make this step a user action, such as "Verify that the version number at the end of the page is correct." Or, consider combining this step with the result in the previous step, "When complete, the status changes to *Succeeded* and *Up to date*, and the end of the page displays the version number."